packages/lesswrong/components/tagging/TagPageButtonRow.tsx

```diff
@@ -97,32 +98,52 @@ const TagPageButtonRow = ({tag, editing, setEditing, className, classes}: {
       e.preventDefault();
     }
   }
-  
+
+  const showEdit = !editing
+  const noEditKarmaTooLow = currentUser && currentUser.karma < tagMinimumKarmaPermissions.edit
+  const noEditNotAuthor = currentUser && !currentUser.isAdmin && tag.authorOnly && tag.userId !== currentUser._id
```

Comment:
Currently this returns a falsey value if not logged in, which probably isn't what you wanted.

Reply:
Separately: if we greenlight an author, it shouldn't matter if they have 0 karma, they should be able to edit. This will matter, as I'm guessing some of Pablo's academics won't have Forum accounts (although who knows if he'll have them edit the Forum directly, still, we can build this "right" anyway.)